website/integrations: Zoho: cleanup (#22701)
goauthentik/authentik: website/integrations/platforms/zoho/index.mdx

- `accounts.zoho.com` is the Zoho Accounts URL for your organization's data center.

:::info
This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
:::

Use the Zoho Accounts URL that matches your organization's data center:

- United States: `https://accounts.zoho.com`
- Australia: `https://accounts.zoho.com.au`
- Europe: `https://accounts.zoho.eu`
- India: `https://accounts.zoho.in`
- China: `https://accounts.zoho.com.cn`
- Japan: `https://accounts.zoho.jp`
- Canada: `https://accounts.zohocloud.ca`
- Saudi Arabia: `https://accounts.zoho.sa`

## Download Zoho metadata file

1. Log in to Zoho Accounts as an administrator at the Zoho Accounts URL for your data center.
2. Navigate to **Organization** > **SAML Authentication**.
3. Click **Download Metadata**. You will require this Zoho metadata file in the next section.

## authentik configuration

To support the integration of Zoho with authentik, you need to create an application/provider pair in authentik.

> ### Create an application and provider in authentik

Reviewer: ```suggestion
### Create an application and provider
```

## Zoho configuration

> 1. Log in to Zoho Accounts as an administrator at the Zoho Accounts URL for your data center.

Reviewer: ```suggestion
1. Log in to Zoho Accounts as an administrator using your Zoho Accounts URL.
```
